### PR TITLE
Feature/customer navigation link

### PR DESCRIPTION
**Pull request by HannesOster**
Added a link to start a google maps navigation to a customer

### CODE REVIEW

File: components/CustomerNavigation/CustomerNavigation.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Container } from "../Buttons/styles";
+import { EditButton } from "../CustomerInfo/styles";
+
+function getGoogleMapsLink(customer) {
+  const { street, location, plz } = customer;
+  const address = `${street}, ${location}, ${plz}`;
```
Comment (CJAkkuec) [inline components/CustomerNavigation/CustomerNavigation.js]:
```suggestion
  const address = `${street}, ${location}, ${plz}`;
```
"plz" should probably be areaCode 🤔

**Review by CJAkkuec — APPROVED** [review]
Approved, because the change I've noted doesn't warrant another code review after the change has been implemented.

File: components/CustomerNavigation/CustomerNavigation.js (hunk 1 above)
Comment (HannesOster) [reply components/CustomerNavigation/CustomerNavigation.js]: Im sorry, but i dont get whats the change in this suggestion? 

Also i dont know what areaCode means :see_no_evil:

Comment (HannesOster) [reply components/CustomerNavigation/CustomerNavigation.js]: ah okay i got it now :)

**Pull request merged**